### PR TITLE
Add GUI for CANDOP

### PR DESCRIPTION
**Pull request by MarcusHolly**
## Summary/Motivation:
Adds GUI for CANDO_P flowsheet

## Changes proposed in this PR:
-Adds GUI file

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Review by adam-a-a — APPROVED** [review]
LGTM - we might want to add the corresponding entry point to the setup.py file, especially if tests fail due to latest PR by @lbianchi-lbl.

**Comment by lbianchi-lbl** [comment]
> LGTM - we might want to add the corresponding entry point to the setup.py file, especially if tests fail due to latest PR by @lbianchi-lbl.

Good point - at the moment, the only CI check that would fail if a UI module is created but not registered in `setup.py` is the Codecov project coverage one, and even then it's not guaranteed as it depends on whether the module has enough uncovered code to surpass the relaxed tolerance (configured at 1%, i.e. PRs are allowed to cause the project coverage to go down to `current_coverage - current_coverage * 0.01`).

Long story short: reviewers should ensure that each UI-enabling module is registered in `setup.py` (and that the corresponding tests all pass; see also #764).

**Review by bknueven — APPROVED** [review]
